ChangeLog.md

```diff
@@ -1,3 +1,109 @@
+## 2.5.0 - July 2019
+#### Az.Accounts
+* Update common code to use latest version of ClientRuntime
+
+#### Az.ApplicationInsights
+* Fix example typo in 'Remove-AzApplicationInsightsApiKey' documentation 
+
+#### Az.Automation
+* Fix typo in resource string 
+
+#### Az.CognitiveServices
+* Added NetworkRuleSet support.
+
+#### Az.ContainerRegistry
```

Comment:
Az.Compute not in release notes, but that is ok

Reply:
I'll fix it. I knew I missed something.